packages/core/test/mocks/integration.ts

```diff
@@ -36,3 +36,13 @@ export class AddAttachmentTestIntegration implements Integration {
     });
   }
 }
+
+export class AdHockIntegration implements Integration {
```

Comment:
sorry for the nit and feel free to ignore this as it's only a test but should this be called `AdHocIntegration`?

Reply:
Thanks, yes, I'll fix it. Hate typos.